File: Packets/HandshakeResponse.vb
Public Class HandshakeResponse
    Inherits Packet
    Public Overrides Property OPCode As Byte = &H8
    Public Property ClaimResponse As String
    Public Property ProcessedHash As String
    Sub New(ByVal Bytes As Byte())
        MyBase.New(Bytes, False, True)
        Dim b = Payload
        ClaimResponse = ReadString()
        ProcessedHash = ReadString()
    End Sub

    Public Overrides Function GetByteArray() As Byte()
        Return Nothing
    End Function
End Class

File: Packets/HandshakeResponse.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class HandshakeResponse : Packet
{
	public override byte OPCode { get; set; }
	public string ClaimResponse { get; set; }
	public string ProcessedHash { get; set; }
	public HandshakeResponse(byte[] Bytes) : base(Bytes, false, true)
	{
		dynamic b = Payload;
		ClaimResponse = ReadString();
		ProcessedHash = ReadString();
	}

	public override byte[] GetByteArray()
	{
		return null;
	}
}
